src/services/mentees.js

```diff
@@ -1209,37 +1235,37 @@ module.exports = class MenteesHelper {
 			if (organization_ids.length !== 0) {
 				additionalFilter = `AND "organization_id" in (${organization_ids.join(',')})`
 			}
-			if (userPolicyDetails.external_mentor_visibility && userPolicyDetails.organization_id) {
-				const externalVisibilityPolicy = userPolicyDetails[extensionColumns[0]]
+			if (getOrgPolicy.external_mentee_visibility_policy && userPolicyDetails.organization_id) {
+				const visibilityPolicy = getOrgPolicy.external_mentee_visibility_policy
 
 				// Filter user data based on policy
 				// generate filter based on condition
-				if (externalVisibilityPolicy === common.CURRENT) {
```

Comment:
@Prajwal17Tunerlabs No need to rewrite the existing code. Please use the same variable